frontend/src/app/testQueue/TestCardForm/TestCardFormReducer.tsx

```diff
@@ -118,6 +117,7 @@ export const testCardFormReducer = (
           prevState.devicesMap.get(payload)?.swabTypes[0].internalId ??
           prevState.specimenId,
         testResults: [],
+        aoeResponses: {},
```

Comment:
noticed that we weren't reseting the AOE answers when we were updating the device type, which caused things like "symptom" to carry over between different devices. Opted to reset that here since I think a new device warrants a reseting of the AOE state